resources.tf

```diff
@@ -0,0 +1,39 @@
+data opentelekomcloud_images_image_v2 debian {
+  name = "Standard_Debian_10_latest"
+}
+
+resource opentelekomcloud_vpc_v1 vpc {
```

Comment:
what about sec groups?

Reply:
Are they required for terraform acceptance tests?